loader.qml

```diff
@@ -1,57 +1,193 @@
+                                rotationScale: mouseWheelScale
+                            }
+                            delegate: Label {
+                                text: locale.standaloneMonthName(index, Locale.ShortFormat)
+                                horizontalAlignment: Text.AlignHCenter
+                                verticalAlignment: Text.AlignVCenter
+                            }
+                        }
+                        Tumbler {
+                            id: daysTumbler
+                            enabled: settime.checked
+                            currentIndex: 24
+                            model: 31
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                            delegate: Label {
+                                text: index + 1
+                                horizontalAlignment: Text.AlignHCenter
+                                verticalAlignment: Text.AlignVCenter
+                            }
+                        }
+                    }
+                }
+                Frame {
+                    padding: 0
+                    Row {
+                        Tumbler {
+                            id: hoursTumbler
+                            enabled: settime.checked
+                            currentIndex: 16
+                            model: 24
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                        }
+                        Tumbler {
+                            id: minutesTumbler
+                            enabled: settime.checked
+                            currentIndex: 58
+                            model: 60
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                        }
+                        Tumbler {
+                            id: secondsTumbler
+                            enabled: settime.checked
+                            currentIndex: 28
+                            model: 60
+                            WheelHandler {
+                                property: "currentIndex"
+                                rotationScale: mouseWheelScale
+                            }
+                        }
+                    }
+                }
             }
-            Item { Layout.fillWidth: true }
-            Button {
-                id: screenshot
-                text: "Screenshot"
-                onClicked: myFrame.snapshot()
+            RowLayout {
+                Text {
+                    leftPadding: 100
+                    text: "featureSlider"
```

Comment:
What does this do ? I don't get it.
Am I missing something or can there be a more descriptive name ?

Reply:
The featureSlider can be used to fake all sorts of input and see the result "in realtime". For the last watchface for instance i used it to simulate the battery display to quickly check whether it displays correctly at all charging percentages.
I had such tooling in separate scripts and welcome the addition into the growing WF simulator.

Reply:
I see :) maybe « custom value » could be a better name than feature ? I don’t know, maybe there are better suggestions

Reply:
I added a tooltip to help explain.  I could also add a menuBar with a Help item, I suppose.  Or would this be better added to the project's .md file?

Reply:
Ah nice, a tooltip is a great idea yes.
Adding a bit of documentation to README.md never hurts ;)

Reply:
Indeed, I've updated the README.md as well!